docs/learn/tutorials/versioned/samza-rest-getting-started.md

```diff
@@ -48,7 +48,7 @@ Run the following commands:
 {% highlight bash %}
 cd samza-rest/build/distributions/
 mkdir -p deploy/samza-rest
-tar -xvf ./samza-rest-1.0.1-SNAPSHOT.tgz -C deploy/samza-rest
+tar -xvf ./samza-rest-1.1.1-SNAPSHOT.tgz -C deploy/samza-rest
```

Comment:
1. Shouldn't this be `1.1.0` instead of `1.1.1` 
2. Why are we using the snapshotted version for `samza-rest`?

Reply:
1. I believe since this is on master it should be the SNAPSHOTTED version of the next release (ie 1.1.1-SNAPSHOT) instead of the latest released version.
2. Following convention of the previous releases, master should be named as a SNAPSHOT to be picked up in maven local